src/main/java/org/folio/rest/impl/PurchaseOrderLineHelper.java

```diff
@@ -160,11 +136,41 @@ private CompositePurchaseOrder validateOrderState(CompositePurchaseOrder po) {
     subObjFuts.add(createReportingCodes(compPoLine, line));
 
     return allOf(subObjFuts.toArray(new CompletableFuture[0]))
+      .thenCompose(v -> setTenantDefaultCreateInventoryValues(compPoLine))
       .thenCompose(v -> generateLineNumber(compOrder))
       .thenAccept(lineNumber -> line.put(PO_LINE_NUMBER, lineNumber))
       .thenCompose(v -> createPoLineSummary(compPoLine, line));
   }
 
+   CompletableFuture<Void> setTenantDefaultCreateInventoryValues(CompositePoLine compPOL) {
+    CompletableFuture<JsonObject> future = new VertxCompletableFuture<>(ctx);
+
+    if ((compPOL.getPhysical() != null && compPOL.getPhysical().getCreateInventory() == null)
+      || (compPOL.getEresource() != null && compPOL.getEresource().getCreateInventory() == null)) {
+      loadConfiguration(okapiHeaders, ctx, logger)
+        .thenApply(config -> future.complete(Optional.of(config.getJsonObject(CREATE_INVENTORY)).orElse(new JsonObject())))
+        .exceptionally(t -> future.complete(new JsonObject()));
+      return future
+        .thenAccept(jsonConfig -> updateCreateInventory(compPOL, jsonConfig));
+    } else return completedFuture(null);
+  }
+
+  private void updateCreateInventory(CompositePoLine compPOL, JsonObject jsonConfig) {
+    //try to set createInventory by values from mod-configuration. If empty - set default hardcoded values
+    if (compPOL.getEresource() != null && compPOL.getEresource().getCreateInventory() == null) {
+      String createInventory = jsonConfig.getString(ERESOURCE);
+      Eresource.CreateInventory eresource = StringUtils.isEmpty(createInventory) ?
```

Comment:
this is a bit of a nitpick but this variable name is poorly named... consider removing this temporary var altogether or renaming it to something like `effectiveValue`

Reply:
variable naming updated